### ISSUE TITLE
Fear Bug

### ISSUE THREAD

**Issue by ohrerbohrer** [FeenixServerProject/Archangel_2.4.3_Bugtracker#3739]
When a Warrior, Lock or Priests fears someone you don't see to where he is feared, so u cannot cast or hit him then. The only one who is able to see the Position is the one who got feared

**Comment by tokash1337** [comment]
You need any further informations about this Issue to fix it? I mean, this is a game breaking Bug (Yes I know there are a lot of them) and it exists for more than 1 1/2 months now. 

Why does it take so long to fix such elementary bugs ? The worst thing is that it seems nobody cares .
A statement for information and processing would be a first step .

**Comment by Fleischi** [comment]
same weirdness with stunned mobs in our raids. they appear to move forward visually but are still attackable at their original position. when the stun runs out the mob warps back to its original position.

**Comment by Urshark** [comment]
Yes, this is realy disappointing... noone cares again. No response.

**Comment by Forcy** [comment]
We are looking into this issue and will be resolved as soon as we can.

**Comment by Urshark** [comment]
Ty, for your answer. Wish you all the best. So one more request here -
https://github.com/FeenixServerProject/Archangel_2.4.3_Bugtracker/issues/3730
Could u tell me if u know about this problem? Only asking to know if I
should stay on feenix or not.

Tyvm

Best regards

Urshark

2016-02-12 15:08 GMT+01:00 Forcy <notifications@github.com>:

**Comment by tokash1337** [comment]
Hi guys,

I've got 3 little examples to show you the problem.

http://www.twitch.tv/tokash1337/v/49120191 

http://www.twitch.tv/tokash1337/v/49123147

https://www.youtube.com/watch?v=jvjzBzF4ha0 at 00:01:43 you can see a good exaple in PvP. I try to attack the feared player where he seems to be visual. But it says " out of range although im standing in range to him visual. In realitiy he is somewhere else.

-This is a serious bug. For me as a warrior in PvP . But Imagine you play Warrlock and you wanna fear someone twice in a row. You face in the feared targets direction and it says "Not in line of sight" 2 Sec later he spawns 15 yards away behind you.

-In my opinion its just a visual bug because I found out that you can hit the feared player/npc if you know where they are actually.

-The Problem is that u cant see where they are getting feared to. They kinda moonwalk and teleport after. 

-From the point of view of the feared player all works fine.

-As far as I know this problem exists with all fear mechanics like Int. Shout/ Fear/ Psychic Scream etc...

Do you need any further evidence or Information from the Sittuation from the live realm? Please let me know I will record it for you guys.

Thanks for your help and
kind regards,

Tokash

**Comment by tokash1337** [comment]
There are multiple possibilities. Either. 

-You dont want to solve this problem. 
-You can't solve this problem.
-Or you dont give a fuck at all.

Just pick one of those answers and close this thread.

**Comment by Nzzoth** [comment]
Hello Tokash1337, I'll pick point number 4.

- Not everything is easy and straight forward as you think.

More over this is a issue report not a topic.

**Comment by teddys** [comment]
https://www.twitch.tv/teddysxyz/v/56627829
18:50
...

**Comment by repae1f** [comment]
still not fixed

**Comment by luvs2spooch** [comment]
seriously? 7 months?
its not always unfair to the warlock/priest/whatever, its unfair for the one getting feared too, the one moment you are next to the caster the other you running at 100% increased speed away from him
this is not a minor glitch, its a major bug

**Comment by repae1f** [comment]
http://www.wow-one.com/forum/topic/105928-community-update/?page=2#comment-767027

**Comment by Zerixx** [comment]
Yo, fixed this in my most recent movement system revamp. Fears, stuns etc... all good :)

Fixed 12-04-16.
